pkgs/servers/owncast/default.nix

```diff
@@ -1,43 +1,34 @@
-{ lib, buildGoModule, fetchFromGitHub, nixosTests, bash, which, ffmpeg, makeWrapper, coreutils, ... }:
-
-buildGoModule rec {
+{
+  lib
+  ,buildGoModule
+  ,callPackage
+  ,fetchFromGitHub
+  ,nixosTests
+  ,ffmpeg
+  ,makeWrapper
+  ,system
+}:
+
+let
+  version = "0.1.1";
```

Comment:
Why are we using a `let ...  in` statement here?
`version` is only being used a single time.
Is there any reason for this change? In my opinion it just makes the code harder to read.

Reply:
see comment on @SuperSandro2000's review below